auslib/test/admin/views/test_permissions.py

```diff
@@ -406,6 +406,14 @@ def testUpdateScheduledChangeExistingPermissionResetSignOffs(self):
         self.assertEquals(len(rows), 0)
 
     @mock.patch("time.time", mock.MagicMock(return_value=300))
+    def testUpdateCompletedScheduledChangeExistingPermission(self):
```

Comment:
Do keep these tests though! It's good to test both the web layer and the database layer.